### PR TITLE
feat(removeTranslations): Delete all things related to translations

### LINKED ISSUE #3425: Make vuestic docs english only
It doesn't seem we want to have translations now or in the future:

Pros:
* right now translations are not full (not perfect to showcase)
* we're targeting international community (translations add to clutter)
* #3039 - search gives multiple results because of translations
* GPTs are on the rise and translations are steadily improving, so it might be very easy to translate full page in a year or two. So no much reason to have translation going forward.

Cons:
* we'll have a cut on russian audience
* we will get rid of translations for good (there won't be a simple way to reenable them)

### Action

- [ ] Remove translation switch from docs page
- [ ] Remove translations from docs (to research and detail - would probably include router, page configs and translation files, but there might be edge cases)

### PR DESCRIPTION
**Pull request by misterpekert**
closes #3425

### CODE REVIEW

File: packages/docs/components/layout/LanguageDropdown.vue [hunk 1]
```diff
@@ -34,10 +34,10 @@
```
Comment (asvae) [inline packages/docs/components/layout/LanguageDropdown.vue:34]: Can't we remove language dropdown altogether?

File: packages/docs/modules/page-config/blocks/api/index.vue [hunk 2]
```diff
@@ -112,10 +109,10 @@ const slotsOptions = computed(() => Object
     name: key,
     description: prop.types
       ? {
-          text: t(getTranslation('slots', key)) + '. ' + t(getTranslation('slots', 'scopeAvailable')),
+          text: getTranslation('slots', key) + '. ' + getTranslation('slots', 'scopeAvailable'),
```
Comment (asvae) [inline packages/docs/modules/page-config/blocks/api/index.vue:112]: Same here. (shouldn't be called `getTranslation`)

File: packages/docs/modules/page-config/blocks/list/index.vue [hunk 3]
```diff
@@ -9,15 +9,13 @@ const props = defineProps({
   }
 })
 
-const { t } = useI18n()
-
-const translateAndMark = (translations: string[]): string => {
+const mark = (translations: string[]): string => {
```
Comment (asvae) [inline packages/docs/modules/page-config/blocks/list/index.vue:12]: These are not translations?

File: packages/docs/modules/page-config/blocks/shared/code/useCode.ts [hunk 4]
```diff
@@ -1,34 +1,34 @@
...
-   */
-  function applyTranslations (code: string) {
-    return translateOthers(translateInterpolation(code, $t), $t).replace('$t-ignore', '$t')
-  }
-
-  return {
-    applyTranslations
-  }
-}
+// export const useCode = () => {
```
Comment (asvae) [inline packages/docs/modules/page-config/blocks/shared/code/useCode.ts:1]: Let's remove it. Was useful before, not now.

File: packages/docs/modules/page-config/i18n/export-translations.ts [hunk 5]
```diff
@@ -5,6 +5,7 @@ import { existsSync } from 'fs'
 
 export const exportTranslations = async (importer: Importer, configPath: string) => {
   const translations = dirname(configPath) + '/translation'
+  console.log(translations);
```
Comment (asvae) [inline packages/docs/modules/page-config/i18n/export-translations.ts:8]: Console.log )

File: packages/docs/page-config/ui-elements/toast/api-description.ts [hunk 6]
```diff
@@ -0,0 +1,23 @@
+export default {
```
Comment (asvae) [inline packages/docs/page-config/ui-elements/toast/api-description.ts:1]: We can use define function here similar to `defineManualApi`. That way we'll have proper typing.

File: packages/docs/pages/[page-config-category]/[page].vue [hunk 7]
```diff
@@ -66,48 +52,48 @@ const compileTranslations = (translations: Record<string, any>): any => {
...
-  if (!configTranslations) { return }
-
-  // extract installation from getting-started/installation
-  const [translationKey] = pageConfigName.value.split('/').slice(-1)
-
-  mergeLocaleMessage(language, {
-    [translationKey]: compileTranslations(configTranslations),
-  })
-}
+// const mergeTranslations = () => {
```
Comment (asvae) [inline packages/docs/pages/[page-config-category]/[page].vue:55]: Let's remove these comments.

**Review by asvae — CHANGES_REQUESTED** [review]
Couple of small things to finish, but very close to done. Great work!

I went through all functional files, and randomly checked about 10% of page-config pages.

File: packages/docs/README.md [hunk 8]
```diff
@@ -33,9 +33,6 @@ We use `@nuxtjs/tailwindcss` plugin.
 Try to use as much as you can less of tailwind in demos that will be visible to user, so demo can be easy copied into user's project
 without Tailwind.
```
[the remark this answers is not in the document]
Comment (misterpekert) [reply packages/docs/README.md:34]: Those used as code examples on `services/i18n` page

File: packages/docs/page-config/ui-elements/affix/index.ts [hunk 9]
```diff
@@ -11,7 +12,7 @@ export default definePageConfig({
     block.example("Default", { title: "Fixes element relative to custom container" }),
 
     block.subtitle("API"),
-    block.api("VaAffix", {
+    block.api("VaAffix", defineApiDescription({
```
Comment (asvae) [inline packages/docs/page-config/ui-elements/affix/index.ts:15]: Probably not needed here. TS can derive type from `block.api` argument typing.

**Pull request merged**